test/topology_custom/test_tablets.py

```diff
@@ -23,3 +23,55 @@ async def test_tablet_change_replication_vnode_to_tablets(manager: ManagerClient
     with pytest.raises(InvalidRequest):
         await cql.run_async("ALTER KEYSPACE test WITH replication = {'class': 'NetworkTopologyStrategy', 'replication_factor': 1, 'initial_tablets': 1};")
 
+
+@pytest.mark.asyncio
+async def test_tablet_change_replication_strategy(manager: ManagerClient):
```

Comment:
I'm confused, why do we have both test/topology_custom/test_tablets.py (a tiny file) and test/topology_experimental_raft/test_tablets.py?
I think you added this test in the wrong file, which is why you had to add the "experimental_features" thing below.

Reply:
I've no idea why there's topology_experimental_raft/test_tablets.py. My goal was to move tablets python test from topology suite (which pre-creates cluster with default options) into some other suite that doesn't and the test could start cluster on its own. The topology_custom seemed like good enough place.